docs/articles/documentation/guides/advanced-guides/test-https-features-and-http2-websites.md

```diff
@@ -23,21 +25,105 @@ If the tested page does not use HTTPS-specific features (like
 [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API),
```

Comment:
I don't know if it the right place to describe it, but all modern browsers thread `localhost` as a secure origin by default. It means that you don't have to mess with certificates if you want to test built-in browser features that require secure origin (Service workers, GeoAPI, haven't tried the CryptoAPI, but should work too). Third-party JS features like ApplePay may or may not work depending on its implementation.